ironfish/src/rpc/routes/node/getStatus.ts

```diff
@@ -205,7 +205,7 @@ function getStatus(node: IronfishNode): GetStatusResponse {
       memTotal: node.metrics.memTotal,
     },
     miningDirector: {
-      status: 'started',
+      status: node.miningManager.minersConnected ? 'started' : 'stopped',
```

Comment:
Now it will say, "Stopped" when the miner director is started and accepting connections from miners. This is not correct now.

Reply:
Hi, NullSolier.
You got it wring. Do you know about ternary operator? 
A ? B : C
If A == true, goto B 
else
  goto C
In my opinion, the fixed code is true.

Reply:
I analyzed this ironfish code for several days.
So I find many source code bugs like this.
If project contains these bugs yet, you must fix these.

Reply:
I'd like to explain ternary operator A ? B : C
If A == true? goto B: else goto C
So in the fixed code when the miner director is started and accepting connections from miners, miningDirector's status is 'started'. When the miner director is not started and not accepting connections from miners, miningDirector's status is 'stopped'.

So this bug must be fixed in this project.